src/tests/class_tests/smartpeak/data/RawDataProcessor_serumTest_accurateMassSearch.featureXML

```diff
@@ -13,7 +13,7 @@
 			<UserParam type="float" name="peak_apex_int" value="4663.75927734375"/>
 			<UserParam type="string" name="PeptideRef" value="2546191730263012646"/>
 			<UserParam type="string" name="used_" value="false"/>
-			<UserParam type="string" name="timestamp_" value="2021-07-01-14-52-30"/>
+			<UserParam type="string" name="timestamp_" value="2021-08-04-15-42-21"/>
```

Comment:
It looks like the `RawDataProcessor` test for accurate mass search maybe writing a file to disk that it should not.

Reply:
yes i think this is a reported bug we need to fix one day